public/local.json

```diff
@@ -0,0 +1,80 @@
+{
+    "result": [
+        {
+          "profile_id": "67596525-9cbc-4a26-8aa4-4d1b7df3813f",
+          "first_name": "Kingston",
+          "last_name": "Morrises",
+          "email": "Kingston.Morris@gmail.com",
+          "country": "U.S.",
+          "state": "Missouri",
+          "city": "Rochester",
+          "current_company": "Microsoft",
+          "current_position": "Data Engineer",
+          "tech_stack": [
+            "Backend",
+            "Design UI/UX",
+            "Career Development"
+          ],
+          "commitment": true,
+          "job_help": false,
+          "industry_knowledge": false,
+          "pair_programming": true,
+          "referred_by": "Meta",
+          "other_info": "anything else may be written here",
+          "validate_status": "pending",
+          "is_active": true,
+          "accepting_new_mentees": true,
+          "created_at": "2022-10-31T19:36:48.231000"
+        },
+        {
+          "profile_id": "8ca860e0-0522-44ed-a330-2dc456895e3d",
+          "first_name": "Rhys",
+          "last_name": "Parker",
+          "email": "Rhys.Parker@gmail.com",
+          "country": "U.S.",
+          "state": "Maryland",
+          "city": "New Town",
+          "current_company": "Meta",
+          "current_position": "Project Manager",
+          "tech_stack": [
+            "Data Science",
+            "iOS",
+            "Android"
+          ],
+          "commitment": true,
+          "job_help": false,
+          "industry_knowledge": false,
+          "pair_programming": false,
+          "referred_by": "Meta",
+          "other_info": "anything else may be written here",
+          "validate_status": "pending",
+          "is_active": true,
+          "accepting_new_mentees": true,
+          "created_at": "2022-10-31T19:36:48.231000",
+          "updated_at": "2021-10-31T19:36:48.231000"
+        },
+        {
+          "profile_id": "dd3c58d1-5978-482a-a75c-2a92978e1565",
+          "first_name": "Benjamin",
+          "last_name": "Wright",
+          "email": "Benjamin.Wright@gmail.com",
+          "country": "U.S.",
+          "state": "Virgin Islands",
+          "city": "New Town",
+          "formerly_incarcerated": true,
+          "underrepresented_group": false,
+          "low_income": true,
+          "convictions": "Infraction, Misdemeanor, Felony",
+          "tech_stack": "Frontend",
+          "job_help": true,
+          "pair_programming": true,
+          "referred_by": "Family",
+          "other_info": "anything else may be written here",
+          "validate_status": "pending",
+          "is_active": true,
+          "in_project_underdog": true,
+          "created_at": "2022-10-31T14:10:22.314000",
+          "updated_at": "2020-10-31T17:36:48.231000"
+        }
+    ]
+}
```

Comment:
adding mock data is a good way to avoid waiting for the BE endpoints to be created, smart move